letsencrypt/display/util.py

```diff
@@ -174,6 +182,21 @@ def checklist(self, message, tags, default_status=True, **unused_kwargs):
         return self.dialog.checklist(
             message, width=self.width, height=self.height, choices=choices)
 
+    def directory_select(self, message, **unused_kwargs):
+        """Display a directory selection screen.
+
+        :param str message: prompt to give the user
+
+        :returns: tuple of the form (`code`, `string`) where
+            `code` - int display exit code
+            `string` - input entered by the user
+
+        """
+        root_directory = os.path.abspath(os.sep)
```

Comment:
I got it from the most upvoted response here; https://stackoverflow.com/questions/12041525/a-system-independent-way-using-python-to-get-the-root-directory-drive-on-which-p